composable/chain.json

```diff
@@ -1,9 +1,9 @@
 {
   "$schema": "../chain.schema.json",
-  "chain_name": "composable",
+  "chain_name": "centauri",
```

Comment:
the chain_name must remain as 'composable' matching the folder name.

Reply:
hi @JeremyParish69 kept both as 'composable' thanks